timebox/notify.py

```diff
@@ -73,11 +73,16 @@ def send_image_file(self, filename):
             return False
 
     def send_message(self, message="", **kwargs):
```

Comment:
Shouldn't we change it to `message=None` ?

Reply:
I was thinking about that as well, but I haven't looked into the docs before.

It seems that the `send_message` method of the `BaseNotificationService` defines `message` as a required positional argument. So I guess there is no need for a default value at all. As it is expected to be supplied no matter what by HASS.

Reply:
Okay!